biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -31,38 +36,237 @@ br {
   display: inline-block;
+  margin-bottom: 1.25rem;
+  margin-left: 1.25rem;
+  margin-right: 1.25rem;
+  margin-top: 1.25rem;
 }
 
 .smallBatteryCells {
   display: inline-block;
-  margin-right: 1.25rem;
   margin-bottom: 1.25rem;
+  margin-left: 1.25rem;
+  margin-right: 1.25rem;
+  margin-top: 1.25rem;
 }
 
 .solarPanels {
   display: inline-block;
+  margin-bottom: 1.25rem;
+  margin-left: 1.25rem;
+  margin-right: 1.25rem;
+  margin-top: 1.25rem;
 }
 
 .startDatePicker {
   display: inline-block;
-  margin-top: 1.25rem;
-  margin-right: 1.25rem;
   margin-bottom: 1.25rem;
+  margin-left: 1.25rem;
+  margin-right: 1.25rem;
+  margin-top: 1.25rem;
 }
 
 .submitButton {
   text-align: center;
 }
 
-.townSize {
+.tooltipBuild {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltipBuild .tooltiptextBuild {
+  visibility: hidden;
+  width: 450px;
+  background-color: rgb(22, 22, 22);
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.tooltipBuild:hover .tooltiptextBuild {
+  visibility: visible;
+}
+
+
+.tooltipEndDate {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltipEndDate .tooltiptextEndDate {
+  visibility: hidden;
+  width: 450px;
+  background-color: rgb(22, 22, 22);
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.tooltipEndDate:hover .tooltiptextEndDate {
+  visibility: visible;
+}
+
+.tooltipHeight {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltipHeight .tooltiptextHeight {
+  visibility: hidden;
+  width: 450px;
+  background-color: rgb(22, 22, 22);
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.tooltipHeight:hover .tooltiptextHeight {
+  visibility: visible;
+}
+
+.tooltipLBC {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltipLBC .tooltiptextLBC {
+  visibility: hidden;
+  width: 450px;
+  background-color: rgb(22, 22, 22);
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.tooltipLBC:hover .tooltiptextLBC {
+  visibility: visible;
+}
+
+.tooltipSBC {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltipSBC .tooltiptextSBC {
+  visibility: hidden;
+  width: 450px;
+  background-color: rgb(22, 22, 22);
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.tooltipSBC:hover .tooltiptextSBC {
+  visibility: visible;
+}
+
+.tooltipSolar {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltipSolar .tooltiptextSolar {
+  visibility: hidden;
+  width: 450px;
+  background-color: rgb(22, 22, 22);
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.tooltipSolar:hover .tooltiptextSolar {
+  visibility: visible;
+}
```

Comment:
This is a perfect place to have one class. and then the different css properties can then be specified

Reply:
Thanks will implement it.